src/app/shared/guards/auth.guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LoginService } from '../../auth/services/login.service';
+import { NavigateService } from '../services/navigate.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AuthGuard implements CanActivate {
+
+    constructor(private loginService: LoginService,
+                private navigateService: NavigateService) { }
```

Comment:
Also I suggest to use code formatters (e.g. prettier) to make your code more clean and readable